### PR TITLE
Refactor code, fix issues, update dependencies

### PR DESCRIPTION
**Pull request by plaskevich**
- Added `.env` functionality
- `clean:data` now runs from JS file directly which makes it many times faster
- Removed `cross-env` as it's not used anymore, module scripts are executed in JS files to avoid issues with variables on Windows
- Updated dependencies and Node version
- Updated `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,43 +2,68 @@
...
+Go to the root of this project, where the `package.json` is located and install all node dependencies:
+
+```
+npm install
+```
+      
+### 3. Set your commercetools project credentials
+
+1. In the root of the project create a file named `.env`
+2. Set the `.env` file with your cridentials:
```
Comment (nkuehn) [inline README.md]: cridentials -> credentials. 

And please add a hint that when creating an API client in Sunrise you can copy / paste the environment variables as-is as an alternative to dotenv.

**Comment by nkuehn** [comment]
It's failing on my empty project though:
```
Importing categories...
child_process.js:671
    throw err;
    ^

Error: Command failed: node_modules/.bin/category-sync import -f data/categories.csv -p XXX  -i XXX -AlnzPCR  -s XXX --sphereHost https://api.sphere.io --sphereAuthHost https://auth.sphere.io
    at checkExecSyncError (child_process.js:631:11)
    at execSync (child_process.js:668:15)
    at Object.<anonymous> (/Users/nkuehn/GitHub/commercetools-sunrise-data/dist/lib/categories.js:46:3)
    at Module._compile (internal/modules/cjs/loader.js:759:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
    at Module.load (internal/modules/cjs/loader.js:628:32)
    at Function.Module._load (internal/modules/cjs/loader.js:555:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:824:10)
    at internal/main/run_main_module.js:17:11
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sunrise@1.0.0 import:categories: `npm run build && node dist/lib/categories.js --import`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sunrise@1.0.0 import:categories script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nkuehn/.npm/_logs/2019-07-30T08_58_25_214Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sunrise@1.0.0 import:data: `npm run import:types && npm run import:channels && npm run import:taxCategories && npm run import:zones && npm run import:projectSetup && npm run import:shippingmethods && npm run import:customers && npm run import:categories && npm run import:products && npm run import:orders`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sunrise@1.0.0 import:data script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nkuehn/.npm/_logs/2019-07-30T08_58_25_264Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sunrise@1.0.0 start: `npm run clean:data && npm run import:data`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sunrise@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nkuehn/.npm/_logs/2019-07-30T08_58_25_311Z-debug.log
```

**Comment by plaskevich** [comment]
> It's failing on my empty project though:
> 
> ```
> Importing categories...
> child_process.js:671
>     throw err;
>     ^
> 
> Error: Command failed: node_modules/.bin/category-sync import -f data/categories.csv -p XXX  -i XXX -AlnzPCR  -s XXX --sphereHost https://api.sphere.io --sphereAuthHost https://auth.sphere.io
>     at checkExecSyncError (child_process.js:631:11)
>     at execSync (child_process.js:668:15)
>     at Object.<anonymous> (/Users/nkuehn/GitHub/commercetools-sunrise-data/dist/lib/categories.js:46:3)
>     at Module._compile (internal/modules/cjs/loader.js:759:30)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
>     at Module.load (internal/modules/cjs/loader.js:628:32)
>     at Function.Module._load (internal/modules/cjs/loader.js:555:12)
>     at Function.Module.runMain (internal/modules/cjs/loader.js:824:10)
>     at internal/main/run_main_module.js:17:11
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! sunrise@1.0.0 import:categories: `npm run build && node dist/lib/categories.js --import`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the sunrise@1.0.0 import:categories script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /Users/nkuehn/.npm/_logs/2019-07-30T08_58_25_214Z-debug.log
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! sunrise@1.0.0 import:data: `npm run import:types && npm run import:channels && npm run import:taxCategories && npm run import:zones && npm run import:projectSetup && npm run import:shippingmethods && npm run import:customers && npm run import:categories && npm run import:products && npm run import:orders`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the sunrise@1.0.0 import:data script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /Users/nkuehn/.npm/_logs/2019-07-30T08_58_25_264Z-debug.log
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! sunrise@1.0.0 start: `npm run clean:data && npm run import:data`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the sunrise@1.0.0 start script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /Users/nkuehn/.npm/_logs/2019-07-30T08_58_25_311Z-debug.log
> ```

Are the URLs in `.env` full or just host name? I'm not sure but if they are full URLs then it's causing the problem. The module we use for importing categories needs only host name (e.g. api.sphere.io) and crashes otherwise

**Comment by nkuehn** [comment]
I did it as documented in the README, having to put only the hostname is an old thing that has not changed in this brach.

**Pull request merged**